docs/zh/start/trace/prometheus.md

```diff
@@ -0,0 +1,38 @@
+# Prometheus metrics 接入
+
+## 运行prometheus
+
+```shell
+cd diagnostics/prometheus
+
+docker-compose -f prometheus-docker-compose.yaml up -d
+```
```

Comment:
Could be help make this doc "runnable" and add it into CI according to https://mosn.io/layotto/#/zh/development/test-quickstart and https://github.com/seeflood/mdx#examples
You can take existing quickstart doc as examples ,e.g. https://github.com/mosn/layotto/blob/main/docs/zh/start/configuration/start.md?plain=1